warrior/Warriorspace/Config_files/Samples/CLI_Test_Data_Sample.xml

```diff
@@ -226,6 +241,14 @@ Also refer the python regex documentation https://docs.python.org/2/library/re.h
 							will be executed in each iteration. This feature is always disabled, user can turn it on by providing "y" and this
 							tag is applicable only for the testdata commands without any iteration pattern in it.
 
+		20. :resp_keys:		(comma separated list of keys) each key corresponds to a pattern which is either mentioned in this testdata section
+							or from the global section.
+
+		21. :inorder_resp_ref:(supported value = "n") patterns to be picked from the response in order or not. By default the patterns are
+							picked in order i.e., if resp_keys is 'k1, k2, k3' then in the response the patterns corresponding to k1 should occur
+							in the beginning followed by k2 and k3. If the inorder_resp_ref is set to 'no' then each of the patterns had to be
+							checked in the whole response every time.
+
```

Comment:
Move up 'resp_keys' & 'inorder_resp_ref' next to 'resp_ref' to group similar options.